articles/Remote Debugging/how-to-remote-debugging-with-vorlonJS.md

```diff
@@ -0,0 +1,33 @@
+<!--
+   tags:
+       Debugging:
+           - "Remote debugging with VorlonJS"
+-->
+# Remote debugging with VorlonJS
+
+> This article explains how to remotely debugging and testing your JavaScript using `vorlon.js`. While working with a Mac and an iPad might be straight forward, achievieving the same thing with a PC is more complicate. Here is one possible approach.
+
+## Server (your local machine)
+
+Install VorlonJS server from npm
+```console
+$ npm i -g vorlon
+```
+
+Then, run the server
+```console
+$ vorlon
+$ The Vorlon server is running
+```
+
+The server is now running and you have the ability to listen to clients.
+To actually see the clients linked with Vorlon, you can go now to Vorlon dashboard by opening `http://localhost:1337/dashboard`
+
+## Client (remote device)
+To link your client page with the Vorlon server, you need to add the following script tag to your page. 
```

Comment:
I needed to add it to the `<head>` - when I had it in the body it was not being initialised correctly.